### ISSUE TITLE
Ledger Live not synchronizing with VeChain Sync.

### ISSUE THREAD

**Issue by LWFM** [LedgerHQ/ledger-live-desktop#3729]
Installed VeChain on Ledger Live. Created a Wallet on VeChain Sync. But when I open Ledger on Sync it hangs at "navigate to VeChain app." Thanks for solution.

**Comment by LWFM** [comment]
Got it working  ... deleted the single wallet created by the center screen icon. 
Created wallets from the top right icons and the first wallet populated from the old deleted Veforge wallet I had.

**Comment by z9finance** [comment]
-Ledger was restored successfully
-Updated firmware
-Updated Vet Wallet
-SYNC is up to date
-Mnemonic keys validated
-I see the old wallet on my computer SYNC wallet
-I see Balance when I view block explorer
I enter the recovery phrase and ledger was restored then I went to the sync and tried to import wallet using the BIP39 - Mnemonic Code I got the list of Addresses at the bottom of the page and the Private keys
inserted the private keys of first address listed (address does not match my original) into SYNC but I get a balance of 0
I cannot find the original address that I can view in the block explorer.
is there a way to find my Original address and retrieve the private keys?

Alternatively when  I try to view the original wallet in Sync I get loop sending me back to "connect and unlock ledger" I get as afar as navigate to Vechain app when I select vechain app on the ledger  loop starts again "connect and unlock ledger"
Before I would get  red caution symbol when I got to checking account info that is when I updated ledger firmware.
